pywr/recorders/calibration.py

```diff
@@ -82,3 +82,84 @@ def values(self):
         return 1.0 - np.sum((obs-mod)**2, axis=0)/np.sum((obs-obs_mean)**2, axis=0)
 
 
+class CoefficientOfDeterminationNodeRecorder(AbstractComparisonNodeRecorder):
+    """ Recorder evaluates the Coefficient of Determination of model and observed """
+    def values(self):
+        mod = self.data        
+        obs = self._aligned_observed
+        n_items = len(mod)    
+        return ((n_items*np.sum(mod*obs, axis=0)-np.sum(mod, axis=0)*np.sum(obs, axis=0))/(np.sqrt((n_items*np.sum(mod**2, axis=0)-(np.sum(mod, axis=0))**2)*(n_items*np.sum(obs**2, axis=0)-(np.sum(obs, axis=0))**2))))**2
```

Comment:
Is it possible to split this calculation over multiple lines to improve readability?